app/views/YouCannotAppealPage.scala.html

```diff
@@ -0,0 +1,68 @@
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import utils.SessionKeys
+
+@this(layout: Layout,
+link: components.link,
+formHelper: FormWithCSRF)
+
+@()(implicit userRequest: UserRequest[_], messages: Messages, appConfig: AppConfig)
+@appealType = @{userRequest.session.get(SessionKeys.appealType)}
+
+@layout(pageTitle = messages("youCannotAppeal.title"),
+isPageFullWidth = false) {
+<h1 class="govuk-heading-l">@messages("youCannotAppeal.heading")</h1>
+    @if(userRequest.isAgent){
+            @if(appealType.contains(PenaltyTypeEnum.Late_Payment.toString)
+                || appealType.contains(PenaltyTypeEnum.Additional.toString)){
+        <p class="govuk-body">@messages("agent.youCannotAppeal.lpp.p1")</p>
+        <p class="govuk-body">@messages("agent.youCannotAppeal.lpp.p2")</p>
+        }  @if(appealType.contains(PenaltyTypeEnum.Late_Submission.toString)){
+        <p class="govuk-body">@messages("agent.youCannotAppeal.lsp.p1")</p>
+        <p class="govuk-body">@messages("agent.youCannotAppeal.lsp.p2")</p>
+            }
+        <p class="govuk-body">@messages("agent.youCannotAppeal.p3")</p>
+        <ul class="govuk-list">
+            <li>
+                <a class="govuk-link" href="@{appConfig.vatOverviewUrl}" id="vat-client-details-link">@messages("agent.youCannotAppeal.returnToVATDetails")</a>
+            </li>
+        </ul>
+    }else {
+        @if(appealType.contains(PenaltyTypeEnum.Late_Payment.toString)
+            || appealType.contains(PenaltyTypeEnum.Additional.toString)){
+        <p class="govuk-body">@messages("youCannotAppeal.lpp.p1")</p>
+        <p class="govuk-body">@messages("youCannotAppeal.lpp.p2")</p>
+        <p class="govuk-body">@messages("youCannotAppeal.lpp.p3")</p>
+        <ul class="govuk-list">
+            <li>
+                <a class="govuk-link" href="@{appConfig.whatYouOwe}" id="check-what-you-owe-link">@messages("youCannotAppeal.checkWhatYouOwe")</a>
+            </li>
+        </ul>
+        }  @if(appealType.contains(PenaltyTypeEnum.Late_Submission.toString)){
+        <p class="govuk-body">@messages("youCannotAppeal.lsp.p1")</p>
+        <p class="govuk-body">@messages("youCannotAppeal.lsp.p2")</p>
+        <p class="govuk-body">@messages("youCannotAppeal.lsp.p3")</p>
+            <ul class="govuk-list">
+                <li>
+                    <a class="govuk-link" href="@{appConfig.vatOverviewUrl}" id="vat-details-link">@messages("youCannotAppeal.returnToVATDetails")</a>
+                </li>
+            </ul>
+            }
+        }
+}
```

Comment:
This can be simplified with our getMessage helper that will figure out if the user is an agent and apply the correct messaging, see:
https://github.com/hmrc/penalties-appeals-frontend/blob/9b90628b31e59e6da8a02a8fb84db56edd7ceef8/app/views/YouCannotAppealPage.scala.html#L28-L52